reference/docs-conceptual/PowerShell-Core-Support.md

```diff
@@ -0,0 +1,69 @@
+# PowerShell Core Support Lifecycle
+
+PowerShell Core is a distinct set of tools and components that ships separately from Windows PowerShell.
```

Comment:
It may be worthwhile to explicitly state that PowerShell Core can be used side-by-side with Windows PowerShell.

Reply:
Do we need to say that here? This is not an install article. This is about support lifecycle. We do talk about side-by-side in the setup articles.

Reply:
I was thinking of simply changing this line to:

```none
PowerShell Core is a distinct set of tools and components that ship separately, but works 
side-by-side with Windows PowerShell.
```

Reply:
I think the message here is about the separateness and the difference in support and licensing. 

@joeyaiello - What do you think?

Reply:
Windows PowerShell is Windows component. So maybe:
```
PowerShell Core is a distinct set of tools and components that ships separately from Windows.
```

Reply:
How about this? 

> PowerShell Core is a distinct set of tools and components that is shipped, installed, and configured separately from Windows PowerShell.

Reply:
I don't understand why we mention `Windows PowerShell`? It is already classic definition that PowerShell Core is a ported, multi-platform product.
>PowerShell Core is a distinct set of tools and components that ships as separate package on Windows and Unix systems. Unlike the Windows PowerShell, the product is not a component of Windows systems and can be installed and configured side-by-side with Windows PowerShell. As a consequence it has its service model which is independent from Windows servicing model.